scripts/install/setup.sh

```diff
@@ -16,6 +16,21 @@ PARENT_DIR=$PARENT_DIR $PARENT_DIR/spinnaker-for-gcp/scripts/manage/check_git_co
 
 source "$PROPERTIES_FILE"
 
+LOCATION_GKE="--zone $ZONE"
+LOCATION_REDIS="--zone $ZONE"
+while getopts ":r" opt; do
+  case ${opt} in
+    r )  echo "Deployment done to a regional cluster in $REGION"
```

Comment:
The console message should look like the other messages. The message is written at the beginning of the installation, when it would feel more natural next to the other cluster-specific and redis-specific messages. Maybe the cluster-specific and redis-specific messages could just be updated instead.
![image](https://user-images.githubusercontent.com/8437718/78618271-f106fe80-7847-11ea-9431-6879da92f6aa.png)